src/Cryptol/REPL/Command.hs

```diff
@@ -183,6 +184,10 @@ nbCommandList  =
     "do type specialization on a closed expression"
   , CommandDescr [ ":eval" ] (ExprArg refEvalCmd)
     "evaluate an expression with the reference evaluator"
+  , CommandDescr [ ":ast" ] (ExprArg astOfCmd)
+    "print out an AST of a given term"
+  , CommandDescr [ ":all" ] (ExprArg allTerms)
```

Comment:
*  The name `all` seems a bit generic, perhaps `ast-all` would work better.
*  Also, it seems wrong to use `ExprArg` as the command has no parameter?